web/stripe-template/api/actions/getProducts.ts

```diff
@@ -0,0 +1,103 @@
+import { stripe } from "../stripe";
+
+type StripeProduct = {
```

Comment:
I think we can get types from Stripe
https://github.com/stripe/stripe-node#usage-with-typescript

Reply:
I tried using this before but it caused more issues